### DISCUSSION TITLE
【已解决】Server端无法启动，请教是怎么回事

### DISCUSSION THREAD

**Discussion by jackiehejian** [apernet/hysteria#706] [General]
在centos上用命令bash <(curl -fsSL https://get.hy2.sh/)部署的，但是服务器端一直启动失败，请教是什么回事
● hysteria-server.service - Hysteria Server Service (config.yaml)
   Loaded: loaded (/etc/systemd/system/hysteria-server.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Thu 2023-09-21 07:47:47 UTC; 5s ago
  Process: 6358 ExecStart=/usr/local/bin/hysteria server --config /etc/hysteria/config.yaml (code=exited, status=1/FAILURE)
 Main PID: 6358 (code=exited, status=1/FAILURE)

Sep 21 07:47:47 localhost.localdomain systemd[1]: Started Hysteria Server Service (config.yaml).
Sep 21 07:47:47 localhost.localdomain systemd[1]: hysteria-server.service: main process exited, code=exited, status=1/FAILURE
Sep 21 07:47:47 localhost.localdomain systemd[1]: Unit hysteria-server.service entered failed state.
Sep 21 07:47:47 localhost.localdomain systemd[1]: hysteria-server.service failed.

**Comment by tobyxdd** [comment]
发 log 和配置

**Reply by jackiehejian** [reply]
已经解决，是没有自签证书的原因，现在已经部署成功，非常感谢，可以close了

**Reply by Rio-de-January** [reply]
我也遇到了同样的问题，一直没有解决，（甲骨文云，ubuntu22.04）

**Comment by jackiehejian** [comment]
我是直接重签证书就解决了。之前就是证书签名不成功导致server无法启动。
















在 2024-04-09 16:27:58，"lyjnb14" ***@***.***> 写道：

请问怎么解决的 我也遇到了同样的问题 自签证书弄了也没有启动成功

**Comment by tanggo-why** [comment]
怎么的，能使详细说一下吗

**Comment by Github-TBH** [comment]
如果使用自己的域名呢？

**Reply by yjd** [reply]
这就不用自签域名，直接acme或hy自带的acme功能

**Reply by yukioyq** [reply]
没有用   用自己的域名 启动不了

**Comment by blog2020** [comment]
如果不用自己的域名，如何自签证书？我看他们都用bing。com，可以用其他网站吗
